cachi2/interface/cli.py

```diff
@@ -102,6 +104,25 @@ class _Input(pydantic.BaseModel, extra="forbid"):
     flags: list[Flag] = list()
 
 
+def _yaml_validate_set_config(file: str) -> str:
+
+    try:
+        with open(file, "r") as value:
+            config.config = config.Config(**yaml.safe_load(value))
+    except yaml.YAMLError:
+        raise typer.BadParameter(f"Not valid YAML: {file!r}")
+    except FileNotFoundError:
+        config.config = config.Config()
+        return ""
```

Comment:
I wouldn't hide non-existent file errors. Rather, we could add `exists=True` to the typer Option to make sure the file exists